src/share/share.validator.ts

```diff
@@ -0,0 +1,41 @@
+import Validator from '../validator';
```

Comment:
Oooooooh....
And I was wondering how the files I've wrote disappeared from my branch!
Now I found them!
Too bad I already have written them again :(

Reply:
Obviously, you need to remove this file from your branch...

Reply:
See? It wasn't just me!